File: NEsper/pattern/guard/Quitable.cs
using System;

namespace net.esper.pattern.guard
{
	/// <summary>
    /// Receiver for quit events for use by guards.
    /// </summary>
	
    public interface Quitable
	{
		/// <summary> Indicate guard quitted.</summary>
		void GuardQuit();
	}
}

File: trunk/esper/src/main/java/net/esper/pattern/guard/Quitable.java
/**************************************************************************************
 * Copyright (C) 2006 Esper Team. All rights reserved.                                *
 * http://esper.codehaus.org                                                          *
 * ---------------------------------------------------------------------------------- *
 * The software in this package is published under the terms of the GPL license       *
 * a copy of which has been included with this distribution in the license.txt file.  *
 **************************************************************************************/
package net.esper.pattern.guard;

/**
 * Receiver for quit events for use by guards.
 */
public interface Quitable
{
    /**
     * Indicate guard quitted.
     */
    public void guardQuit();
}
